trace-test-application/src/androidTest/java/io/bitrise/trace/testapp/screen/BaseScreen.java

```diff
@@ -65,12 +68,21 @@ public UiObject2 find(@NonNull final BySelector by) {
     }
 
     /**
-     * Performs a click action on the given {@link BySelector}.
+     * Performs a click action on the given {@link BySelector}. Retries the finding 3 times.
      *
      * @param by the given BySelector.
      */
     public void click(@NonNull final BySelector by) {
-        find(by).click();
+        for (int i = 0; i < 3; i++) {
```

Comment:
nit: this findCount could be a constant - so that it can more easily be changed in the future?

Reply:
The button will be clicked exactly zero times (when not found) or once, because when it is found and clicked the for loop will terminate with the `break`.

Added the number as a constant to the class.

Reply:
Ah gotcha. I didn't see that in the small snippet window (which would explain why i didn't ask about it yesterday :sweat_smile: ) thank you for your clarification!